packages/kbn-unified-field-list/src/hooks/use_grouped_fields.ts

```diff
@@ -107,6 +107,13 @@ export function useGroupedFields<T extends FieldListItem = DataViewField>({
     // if field existence information changed, reload the data view too
   }, [dataViewId, services.dataViews, setDataView, hasFieldDataHandler]);
 
+  const { allFieldsModified, hasNewFields } = useNewFields<T>({
```

Comment:
I was thinking to call `useNewFields(...)` before `useGroupedFields({ allFields: allFieldsModified, ...})`.

Reply:
but then I would also need to migrate `useExistingFieldsReader` to be called before  `useGroupedFields`? since it's needed by `useExistingFieldsReader`

Reply:
Okay, I see

Reply:
needed by `useNewFields` ... I wanted to say ... changed